conf/munin-docker.config

```diff
@@ -0,0 +1,45 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for Munin Docker
+ *  https://github.com/nextflow-io/nextflow/issues/572
+ * -------------------------------------------------
+ */
+
+docker {
+  enabled = true
```

Comment:
We have a main [munin.config](https://github.com/nf-core/configs/blob/master/conf/munin.config) at `nfcore/configs`. We have to keep this one because of the defined reference paths. In order to keep the `munin.config` general, we have to define these paths in separate configurations.

Reply:
I know - sorry, my comment wasn't very clear. I was hoping that in nf-core/configs you can have a munin config file that covers the general items such as `docker` and `singularity` setup (as discussed there). Then in this repo a profile called `munin` could _also_ exist with just the pipeline specific items, such as references. I'm not sure if the same profile is described twice, but I'm hoping that they're merged. Then this config can be as short as possible and `-profile munin` or `-profile munin,singularity` will both still work.

Make sense?